diff --git a/yolox/utils/ema.py b/yolox/utils/ema.py
index 7b1b8b9..ef1f4d6 100644
--- a/yolox/utils/ema.py
+++ b/yolox/utils/ema.py
@@ -12,13 +12,20 @@ __all__ = ["ModelEMA", "is_parallel"]
 
 def is_parallel(model):
     """check if model is in parallel mode."""
-    import apex
-
     parallel_type = (
         nn.parallel.DataParallel,
         nn.parallel.DistributedDataParallel,
-        apex.parallel.distributed.DistributedDataParallel,
     )
+
+    try:
+        import apex
+    except ImportError:
+        pass
+    else:
+        parallel_type = parallel_type + (
+            apex.parallel.distributed.DistributedDataParallel,
+        )
+
     return isinstance(model, parallel_type)
 
 
